File: TransCoder_tokenized_test_set_functions/java/PROGRAM_SORT_STRING_DESCENDING_ORDER.java
static void sortString ( String str ) {
  int charCount [ ] = new int [ MAX_CHAR ];
  for ( int i = 0;
  i < str . length ( );
  i ++ ) {
    charCount [ str . charAt ( i ) - 'a' ] ++;
  }
  for ( int i = MAX_CHAR - 1;
  i >= 0;
  i -- ) {
    for ( int j = 0;
    j < charCount [ i ];
    j ++ ) {
      System . out . print ( ( char ) ( 'a' + i ) );
    }
  }
}

File: TransCoder_tokenized_test_set_functions/python/PROGRAM_SORT_STRING_DESCENDING_ORDER.py
def sortString ( str ) :
    charCount = [ 0 ] * MAX_CHAR;
    
    for i in range ( len ( str ) ) :
        charCount [ ord ( str [ i ] ) - ord ( 'a' ) ] += 1;
        
    for i in range ( MAX_CHAR - 1, - 1, - 1 ) :
        for j in range ( charCount [ i ] ) :
            print ( chr ( 97 + i ), end = "" );
            
    
